src/CMakeLists.txt

```diff
@@ -232,6 +232,11 @@ install( FILES ${CMAKE_BINARY_DIR}/schema.xsd
 ################################
 # Add python environment setup
 ################################
+
+message(WARNING "Temporarily changing the geosPythonBranch to feature/sherman/baselineStorage")
+set(GEOS_PYTHON_PACKAGES_BRANCH "feature/sherman/baselineStorage" CACHE STRING "" FORCE)
```

Comment:
```suggestion
# Use these commands to set a specific branch of the python packages. 
# message(WARNING "Temporarily changing the geosPythonBranch to feature/sherman/baselineStorage")
# set(GEOS_PYTHON_PACKAGES_BRANCH "feature/sherman/baselineStorage" CACHE STRING "" FORCE)
```

Reply:
Once new baselines have been uploaded and once the pygeosx pr has been merge commit these changes.